requirements.txt

```diff
@@ -22,6 +22,7 @@ elasticsearch-dsl
 certifi
 
 # Core
+enum-compat
```

Comment:
nop, that breaks python3 .. the requirements has been added ton setup.py, you should not have to put it in requirements.txt

Reply:
Ok, then either `CONTRIBUTING.md` either `scripts/env.sh` are not up to date. I had to install `enum34` manually.

Reply:
`pip install -e .` should install it, can you check ?

Reply:
Running it manually works. 
I'll check what happened...